src/Components/CreateImageWizard/steps/stepMapper.js

```diff
@@ -0,0 +1,15 @@
+export default ({ 'target-environment': targetEnv, release } = {}, skipFirst, skipSecond) => {
```

Comment:
Can you explain why the nextStep() callback is handled this way? It isn't clear to me why skipFirst and skipSecond are neccessary.

Reply:
This is to determine which step is active, I've updated the code to be more straight forward.